docs/source/development/eeps.md

```diff
@@ -1,28 +1,14 @@
 # EEPs
 
-Large changes to estimagic can be proposed in estimagic enhancement proposals, short
-EEPs. They serve the purpose of summarising discussions
-that may happen in chats, issues, pull requests, in person, or by any other means.
-Simple extensions (like adding new optimizers) do not need to be discussed with such
-a formal process.
-
-If the author of an EEP feels that it is ready to be accepted they need to make a
-post in our [zulip workspace](https://ose.zulipchat.com) and a comment on the PR that
-contains the following information:
-
-1. Summary of all contentious aspects of the EEP and how they have been resolved
-2. Every interested party has seven days to comment on the PR proposing the EEP,
-   either with approval or objections.
-3. If there are no unresolved objections after seven days, the EEP will automatically
-   be accepted.
-
-Note that the Pull Requests that actually implement the proposed enhancements still
-require a standard review cycle.
+Estimagic Enhancement Proposals (EEPs) can be used to discuss and design large
+changes. EEP-00 details the EEP process, the estimagic governance model and the
+estimagic Code of Conduct. It is the only EEP that gets continuously updated.
 
 These EEPs are currently in place:
 
 ```{toctree}
 :maxdepth: 1
 
+eep-00-governance-model.md
```

Comment:
I'm not sure it is mentioned anywhere, where the code of conduct will be featured once the EEP is adopted? I suggest moving it from the EEP's to the Development section in the documentation.

Reply:
Very good point! I added it as first item in the development section.